src/vs/workbench/parts/extensions/electron-browser/extensionTipsService.ts

```diff
@@ -555,86 +636,84 @@ export class ExtensionTipsService extends Disposable implements IExtensionTipsSe
 		});
 	}
 
-	private _suggestWorkspaceRecommendations(): TPromise<any> {
+	private _suggestWorkspaceRecommendations(allRecommendations: string[]): void {
```

Comment:
Does this parameter need to be passed in? Can we not use `this._allWorkspaceRecommendedExtensions` directly?